Source/Private/OnlineSubsystemDriftTypes.h

```diff
@@ -85,38 +85,13 @@ class ONLINESUBSYSTEMDRIFT_API FOnlineSessionInfoDrift : public FOnlineSessionIn
 
 using DriftID = uint32;
 
+typedef TSharedPtr<class FUniqueNetIdDrift> FUniqueNetIdDriftPtr;
+typedef TSharedRef<class FUniqueNetIdDrift> FUniqueNetIdDriftRef;
+
 class FUniqueNetIdDrift : public FUniqueNetId
```

Comment:
Looks like the base class now implements `GetTypeHash(other)` with `other.GetTypeHash()` and then `GetTypeHash()` is implemented on each subclass, as opposed to each subclass implementing `GetTypeHash(other)`